.github/workflows/integration_tests/no-data-loss/consume_kinesis.sh

```diff
@@ -1,19 +1,21 @@
 #!/bin/bash
 
 streamName=$1
+input_file=$2
 
-export PYTHONWARNINGS="ignore:Unverified HTTPS request"
+export AWS_ACCESS_KEY_ID=foobar
+export AWS_SECRET_ACCESS_KEY=foobar
 
-shard=$(aws --endpoint-url=http://localhost:4566 kinesis describe-stream --stream-name $streamName --output text | grep SHARDS | awk '{print $2}')
-iterator=$(aws --endpoint-url=http://localhost:4566 kinesis get-shard-iterator --stream-name $streamName --shard-id $shard --shard-iterator-type TRIM_HORIZON --output text)
-lines=$(aws --endpoint-url=http://localhost:4566 kinesis get-records --shard-iterator $iterator --output text | tail -n +2|awk '{print $3}' & sleep 1; kill $!)
+shard=$(aws --endpoint-url=http://localhost:4566 kinesis describe-stream --stream-name $streamName --output text --region eu-central-1 | grep SHARDS | awk '{print $2}')
+
+iterator=$(aws --endpoint-url=http://localhost:4566 kinesis get-shard-iterator --stream-name $streamName --shard-id $shard --shard-iterator-type TRIM_HORIZON --output text --region eu-central-1)
+lines=$(aws --endpoint-url=http://localhost:4566 kinesis get-records --shard-iterator $iterator --output text --region eu-central-1 | tail -n +2 | awk '{print $3}' & sleep 1 ; kill $!)
 
 while read p; do
```

Comment:
I had to revert back the logic of this test. It now looks in the data.txt file, and looks for each line in Kinesis.

The reason for this is that the output of `get-records` is really hard to parse and extract the actual data that was sent to the collector, so this makes it easier for further tests to be implemented.

IMO this logic should be fine too, but if it's needed I'll find a way to use the reverse logic.